src/include/sof/audio/module_adapter/module/modules.h

```diff
@@ -44,22 +45,27 @@ struct comp_dev *modules_shim_new(const struct comp_driver *drv,
 				  const struct comp_ipc_config *config,
 				  const void *spec);
 
-#define DECLARE_DYNAMIC_MODULE_ADAPTER(comp_dynamic_module, mtype, uuid, tr) \
-do { \
-	(comp_dynamic_module)->type = mtype; \
-	(comp_dynamic_module)->uid = SOF_RT_UUID(uuid); \
-	(comp_dynamic_module)->tctx = &(tr); \
-	(comp_dynamic_module)->ops.create = modules_shim_new; \
-	(comp_dynamic_module)->ops.prepare = module_adapter_prepare; \
-	(comp_dynamic_module)->ops.params = module_adapter_params; \
-	(comp_dynamic_module)->ops.copy = module_adapter_copy; \
-	(comp_dynamic_module)->ops.cmd = module_adapter_cmd; \
-	(comp_dynamic_module)->ops.trigger = module_adapter_trigger; \
-	(comp_dynamic_module)->ops.reset = module_adapter_reset; \
-	(comp_dynamic_module)->ops.free = module_adapter_free; \
-	(comp_dynamic_module)->ops.set_large_config = module_set_large_config;\
-	(comp_dynamic_module)->ops.get_large_config = module_get_large_config;\
-	(comp_dynamic_module)->ops.get_attribute = module_adapter_get_attribute; \
-} while (0)
+static inline void declare_dynamic_module_adapter(struct comp_driver *drv,
```

Comment:
Why this change? Looks good itself, but I'm failing to make the conection to the git commit description.

Reply:
this makes excluding `.cmd`  for IPC4 easy